frontend/src/components/PasswordTextField/index.tsx

```diff
@@ -1,139 +1,97 @@
+                })}
+              />
+            )}
+          </ListItemIcon>
+          <ListItemText primary={validator.message} />
+        </ListItem>
       ))}
     </List>
   );
 };
 
 const PasswordTextField = forwardRef<
   HTMLInputElement,
-  TextFieldProps & { displayTooltip?: boolean }
+  TextFieldProps & { displayTooltip?: boolean; input?: string }
 >((props, ref) => {
   const [showPassword, setShowPassword] = useState<boolean>(false);
-  const { displayTooltip = false } = props;
-  const errorMessages = [
-    PASSWORD_MIN_LENGTH_ERROR_MESSAGE,
-    PASSWORD_LOWER_CASE_ERROR_MESSAGE,
-    PASSWORD_UPPER_CASE_ERROR_MESSAGE,
-    PASSWORD_DIGIT_ERROR_MESSAGE,
-    PASSWORD_SPECIAL_CHAR_ERROR_MESSAGE,
-  ];
-  // const validatePasswordError = (
-  //   passwordVal: boolean,
-  //   password: string
-  // ): boolean => {
-  //   return passwordVal
-  //     ? password.length < 8 ||
-  //         !/[a-z]/.test(password) ||
-  //         !/[A-Z]/.test(password) ||
-  //         !/\d/.test(password) ||
-  //         // eslint-disable-next-line no-useless-escape
-  //         !/[ `!@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?~]/.test(password)
-  //     : false;
-  // };
-
-  // const comparePasswordError = (
-  //   isMatch: boolean,
-  //   password: string,
-  //   passwordToMatch: string | undefined
-  // ): boolean => {
-  //   return isMatch ? password != passwordToMatch : false;
-  // };
-
-  // const checkEmptyError = (password: string): boolean => {
-  //   return !password;
-  // };
-
-  // const isInvalid =
-  //   validatePasswordError(passwordVal, password) ||
-  //   comparePasswordError(isMatch, password, passwordToMatch) ||
-  //   checkEmptyError(password);
-
-  // //to listen to other password input changes
-  // useEffect(() => {
-  //   setValidity(
-  //     !(
-  //       validatePasswordError(passwordVal, password) ||
-  //       comparePasswordError(isMatch, password, passwordToMatch) ||
-  //       checkEmptyError(password)
-  //     )
-  //   );
-  // }, [passwordVal, isMatch, password, passwordToMatch, setValidity]);
-
-  // const [showPassword, setShowPassword] = useState<boolean>(false);
-
-  // const handleClickShowPassword = () => setShowPassword((show) => !show);
-
-  // const handleMouseDownPassword = (
-  //   event: React.MouseEvent<HTMLButtonElement>
-  // ) => {
-  //   event.preventDefault();
-  // };
-
-  // const handleMouseUpPassword = (
-  //   event: React.MouseEvent<HTMLButtonElement>
-  // ) => {
-  //   event.preventDefault();
-  // };
-
-  // const handlePasswordChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-  //   const val = event.target.value;
-  //   setPassword(val);
-  //   setValidity(
-  //     !(
-  //       validatePasswordError(passwordVal, val) ||
-  //       comparePasswordError(isMatch, val, passwordToMatch) ||
-  //       checkEmptyError(val)
-  //     )
-  //   );
-  // };
+  const { displayTooltip = false, input = "", ...rest } = props;
 
   return (
     <Tooltip
```

Comment:
is it possible to keep the tooltip open when the user is typing ? because currently the user has to hover over the input field to see the tooltip

![Screenshot 2024-10-04 at 10 16 31 AM](https://github.com/user-attachments/assets/d2f56d8a-8bc7-4d86-96c7-2bbe7f3ff7a0)

Reply:
Good point, I'll look into that